applications/FluidDynamicsApplication/tests/TwoFluidDarcyTest/reference_darcy_density.csv

```diff
@@ -0,0 +1,447 @@
+#ID, PRESSURE
```

Comment:
just as a suggestion, wouldn't it make sense to use the check process that Vicente did? it has the mechanism to verify this

Reply:
We'd still need to store the reference values somehow...
Something that might be smarter is to compare the pressure drop to theoretical (which could be done with just two points) avoiding the reference files altogether.